src/compiler/watchUtilities.ts

```diff
@@ -257,6 +257,30 @@ namespace ts {
         Full
     }
 
+    /**
+     * Updates the map of extended config file watches with a new set of extended config files from a base config file
+     */
+    export function updateExtendedConfigFilesMap(
+        configFile: TsConfigSourceFile,
+        extendedConfigFilesMap: ESMap<string, FileWatcher>,
+        createExtendedConfigFileWatch: (extendedConfigPath: string) => FileWatcher,
+    ) {
+        const extendedSourceFiles = configFile.extendedSourceFiles || emptyArray;
+        // TODO(rbuckton): Should be a `Set` but that requires changing the below code that uses `mutateMap`
+        const newExtendedConfigFilesMap = arrayToMap(extendedSourceFiles, identity, returnTrue);

```

Comment:
Key needs to be Path, value needs to be file name to watch

Reply:
This doesnot create the key with `Path`

Reply:
Ah, I missed the meaning of this the first time around. I'll pass something to allow converting from string to `Path`.